source/bootcamp_content/debuggers.rst

```diff
@@ -0,0 +1,296 @@
+Development Tools and Debuggers
+===============================
+
+.. code-block:: none
+
+    (How to( find( the missing parenthesis)).
+
+
+Topics
+------
+
+* Code Analysis (conceptual)
+* Debuggers
+    * CLI tools
+    * Web Consoles
+* Linters
+    * Style Guides
+* Testing
+    * Code Coverage
+* Virtual Environments
+* Integrated Development Environments
+
+
+Code Analysis
+-------------
+
+======================================  ======================================
+Static                                  Dynamic
+======================================  ======================================
+- Linters                               - Debuggers
+- Type checkers                         - Memory profiling tools
+|                                       - Coverage testers
+======================================  ======================================
+
+Deugging
+--------
+
+     "...a computer program that is used to test and debug other programs..."
+
+Debuggers do what you do already:
+
+* Print (broken) variables.
+* Read and reports error messages.
+* Highlight (incorrect) syntax.
+
+Just very fast very well.
+
+.. figure:: /static/youtube-error.jpg
```

Comment:
I would move this to the top of the slide and right-align it so that everything fits more nicely